packages/react-components/package.json

```diff
@@ -75,7 +75,7 @@
     "glob": "9.3.5",
     "husky": "5.2.0",
     "is-docker": "2.1.1",
-    "karma": "6.3.16",
+    "karma": "6.4.4",
```

Comment:
Karma was updated from 6.3.16 → 6.4.4 to pull in socket.io 4.7.2 (upgraded from 4.2.0) which addressed WebSocket denial-of-service vulnerabilities